cobbler/tftpgen.py

```diff
@@ -266,24 +266,19 @@ def write_all_system_files(self, system, menu_items):
 
             # for tftp only ...
             if working_arch in [Archs.I386, Archs.X86_64, Archs.ARM, Archs.AARCH64,
-                                Archs.PPC64LE, Archs.PPC64EL]:
+                                Archs.PPC64, Archs.PPC64LE, Archs.PPC64EL]:
                 # ToDo: This is old, move this logic into item_system.get_config_filename()
                 pass
 
-            elif working_arch in [Archs.PPC, Archs.PPC64]:
-                # Determine filename for system-specific bootloader config
-                filename = "%s" % system.get_config_filename(interface=name).lower()
+            # FIXME why is this ppc-specific code needed at all?
+            # we can boot ppc64le which never executed this
+            elif working_arch in [Archs.PPC]:
```

Comment:
Hmm, maybe this elif part should get removed altogether.

[Archs.PPC] should be moved up to the other archs.

At some point of time, this all should be more coded like:
Iterate through all supported/set boot_loaders (which may have been set by user and are restricted by arch).

I'd vote for remove the elif working_arch in [Archs.PPC]:

and keep the:
  else:
      pass
for the undefined once in the end.

Reply:
removed